Assets/Scripts/Construction/PlayerConstructionHandler.cs

```diff
@@ -46,6 +47,11 @@ private void ExecutePlacement()
                 ExitPlacement();
                 return;
             }
+            if (isPlacementPositionValid == false)
```

Comment:
The name of the function `IsPlacementPositionValid()` implies that it's going to return a bool. Since this is the only place this value is used anyway, it'd be more intuitive to simply modify the function to return a bool and place it here
```suggestion
            if (IsPlacementPositionValid() == false)
```